MIT.md

```diff
@@ -1,7 +1,4 @@
-## Copyright 2021, [YOUR NAME]
-
-###### Please delete this line and the next one
-###### APP TYPE can be a webpage/website, a web app, a software and so on
+# Belden Mugauri 
 
 Permission is hereby granted, free of charge, to any person obtaining a copy of this [APP TYPE] and associated documentation files, to deal in the [APP TYPE] without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the [APP TYPE], and to permit persons to whom the [APP TYPE] is furnished to do so, subject to the following conditions:
```

Comment:
- [ ] Your MIT License file was not well generated. Please fix the MIT license using this [link](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). 
- [ ] The MIT License is a popular open-source license that allows others to use, modify, and distribute your code as long as they give credit to you the original author, and include the original license text. You can use this link to learn how to add one to your project :wink: This is why we add them to projects so upon completion add a link to your License so it can be accessed by anyone who clicks on the link. :100: